src/Uno.UI/UI/Xaml/DependencyObjectStore.Binder.cs

```diff
@@ -32,13 +34,13 @@ public partial class DependencyObjectStore
 		private readonly object _gate = new object();
 
 		private readonly Dictionary<DependencyProperty, int> _childrenBindableMap = new Dictionary<DependencyProperty, int>(0, DependencyPropertyComparer.Default);
-		private readonly List<object> _childrenBindable = new List<object>();
+		private readonly List<object?> _childrenBindable = new List<object?>();
 
 		private bool _isApplyingTemplateBindings;
 		private bool _isApplyingDataContextBindings;
 		private bool _bindingsSuspended;
-		private DependencyProperty _dataContextProperty;
-		private DependencyProperty _templatedParentProperty;
+		private DependencyProperty _dataContextProperty = UIElement.DataContextProperty;
```

Comment:
```suggestion
		// Pre-initialize to the most likely value, since in practice most DependencyObjects will be UIElements. Note this is not readonly since there is not a single DataContextProperty (since DependencyObject is an interface in Uno)
		private DependencyProperty _dataContextProperty = UIElement.DataContextProperty;
```

I propose a comment, since it took me a while to figure out why this shouldn't be `static readonly` 😛

Reply:
That's fair :) Will add!

Reply:
In theory, it may not need to be assigned, but since it's a non-nullable reference, adding a default value is the workaround. No sure I should keep this that way, though...

Reply:
It's not sufficient to ensure it's initialized to a non-null value in the ctor? I haven't played around with the nullability checking much.

Reply:
Looking at it again, it's not needed. The `InitializeBinder` method is not needed anymore and those variables can be initialized from the ctor.